src/main/java/mate/jdbc/Main.java

```diff
@@ -1,7 +1,25 @@
 package mate.jdbc;
 
+import mate.jdbc.lib.Injector;
+import mate.jdbc.model.Driver;
+import mate.jdbc.service.DriverService;
+
 public class Main {
+    private static final Injector injector = Injector.getInstance("mate.jdbc");
+
     public static void main(String[] args) {
-        // test your code here
+        DriverService driverService = (DriverService) injector.getInstance(DriverService.class);
+        Driver driverOne = new Driver(null, "One", "1234");
+        Driver driverTwo = new Driver(null, "Two", "qwerty");
+        Driver driverThree = new Driver(null, "Three", "asd56");
```

Comment:
Let's overload a constructor if you need a instance without id

Reply:
✅